Sources/Kitura/SwaggerGenerator.swift

```diff
@@ -449,9 +449,12 @@ struct SwaggerDocument: Encodable {
     func isDictEncodedAsTuple(_ type: Any) -> Bool {
         let typeStr = "\(type)"
         let pattern = "^\\(Optional\\(Swift\\.String\\), Optional\\(Swift\\.[a-zA-Z0-9]+\\)\\)$"
-        if let regex = try? NSRegularExpression(pattern: pattern, options: []) {
+        do {
+            let regex = try NSRegularExpression(pattern: pattern, options: [])
```

Comment:
Because this is creating a fixed, known valid regular expression, I think it would be fine to `try!` actually. If this fails, Foundation is broken.

Reply:
Agreed